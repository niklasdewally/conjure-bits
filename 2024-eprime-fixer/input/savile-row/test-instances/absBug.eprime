language ESSENCE' 1.0

find a : int(16..394)
find b : int(16..792)

such that 

|a-b| > 424
