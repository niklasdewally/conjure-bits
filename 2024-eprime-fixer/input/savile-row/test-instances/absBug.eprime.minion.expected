MINION 3

#This file has been automatically generated by TAILOR v0.3.3
#bug-reports: andrea@cs.st-and.ac.uk

**VARIABLES**
BOUND a {16..394}
BOUND b {16..792}

# auxiliary variables
BOUND aux0 {-776..378}
BOUND aux1 {0..776}

# aliases

**SEARCH**

PRINT [
[a],[b]]

VARORDER [
a,b,
aux0,aux1]


**CONSTRAINTS**

weightedsumgeq([1,-1], [a,b], aux0)
weightedsumleq([1,-1], [a,b], aux0)
abs(aux1,aux0)
ineq(424,aux1,-1)

**EOF**
